src/helix-ui/styles/core/hx-tabpanel.less

```diff
@@ -0,0 +1,15 @@
+hx-tabpanel {
+  &:extend(hx-reveal all);
+  background-color: white;
+  overflow-y: auto;
+  padding: 1.5rem 1.25rem;
+
+  &[open] {
+    display: block;
+  }
+}
+
+hx-tabpanel {
+  -ms-grid-row: 1;
+  -ms-grid-column: 1;
+}
```

Comment:
Check to see if this needs the `@support` call. Also, add comments

Reply:
added comments -- because we do `display: flex;` in `hxtabcontent.less`, not needed